daemon/container.c

```diff
@@ -2526,6 +2560,21 @@ container_update_config(container_t *container, uint8_t *buf, size_t buf_len, ui
 		ret = container_config_write(conf);
 		container_config_free(conf);
 	}
+
+	// Wipe container if USB token serial changed
+	if (container_has_token_changed(container, conf)) {
+		if (container_wipe(container)) {
+			ERROR("Failed to wipe user data. Setting container state to ZOMBIE");
+			container_set_state(container, CONTAINER_STATE_ZOMBIE);
+		}
+		if ((container->token.type == CONTAINER_TOKEN_TYPE_USB) &&
+		    smartcard_release_pairing(container)) {
+			ERROR_ERRNO(
```

Comment:
use errno macros only directly after syscalls not after wrapper functions

Reply:
Done